log-router/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.serviceMonitor.enabled }}
```

Comment:
@evilrussian should we also check for prometheusEnabled here with serviceMonitor?

Reply:
Yes. If prometheusOperator is not installed, then CRD ServiceMonitor will not be defined in cluster, and you will get an error 
`error: unable to recognize "ServiceMonitor.yaml": no matches for kind "ServiceMonitor" in version "monitoring.coreos.com/v1`